application/templates/backup.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if (.Values.backup).enabled }}
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: {{ .Values.backup.backupCRName }}
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+    - {{ .Values.backup.includedNamespaces | quote }}
```

Comment:
@ayeshasiddiqa161 shouldn't this be fixed to current namespace? why user has to input it?

Reply:
Resolved